core/src/test/java/org/elasticsearch/index/seqno/GlobalCheckpointSyncActionTests.java

```diff
@@ -83,6 +83,10 @@ public void testTranslogSyncAfterGlobalCheckpointSync() throws Exception {
         final IndexShard indexShard = mock(IndexShard.class);
         when(indexService.getShard(id)).thenReturn(indexShard);
 
+        final ShardId shardId = mock(ShardId.class);
```

Comment:
why not `final ShardId shardId = new ShardId(index, id);` and remove the next line?